examples/project/minute_parity/test_1.py

```diff
@@ -22,6 +22,7 @@ def test_dut_info(module_log: logging.Logger):
 
 
 @pytest.mark.case_name("Test stand info")
+@pytest.mark.case_dependency("test_1::test_dut_info")
```

Comment:
Why do we need to depend on `test_dut_info` if this test is always `True`?

Reply:
deleted dependency